site/src/pages/components/invokers.explainer.mdx

```diff
@@ -384,12 +384,10 @@ that this list is ordered and higher rules take precedence:
 | `<select>`      | `'showPicker'`        | Open the `<select>` option picker UI                                                 |
 | `<input>`       | `'auto'`              | Call `.showPicker()` on the invokee                                                  |
 | `<input>`       | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
-| `<video>`       | `'auto'`              | Toggle the `.playing` value                                                          |
```

Comment:
Per #906 removing auto from video/audio for now we can re-evalute before we ship